cms/djangoapps/contentstore/views/item.py

```diff
@@ -574,7 +574,9 @@ def _duplicate_item(parent_usage_key, duplicate_source_usage_key, user, display_
         duplicate_metadata = {}
         for field in source_item.fields.values():
             if field.scope == Scope.settings and field.is_set_on(source_item):
-                duplicate_metadata[field.name] = field.read_from(source_item)
+                # Do not duplicate 'discussion_id' TNL-1754
+                if field.name != 'discussion_id':
```

Comment:
@symbolist: I would probably declare it as a keyword argument in the field definition (which gets stored into the `Field.runtime_options` dictionary: https://github.com/edx/XBlock/blob/master/xblock/fields.py#L309-L324)

Why is it a problem that the discussions have the same `discussion_id`, though? That's just the pointer into the comments_service, right? Isn't it reasonable to have multiple places in the course reference the same discussion?